kinetic/distribution.yaml

```
@@ -5028,6 +5028,21 @@ repositories:
       url: https://github.com/astuff/ibeo_lux.git
       version: release
     status: developed
+  ifm3d_core:
+    doc:
+      type: git
+      url: https://github.com/ifm/ifm3d.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/ifm/ifm3d-release.git
+      version: 0.17.0-9
+    source:
```

```suggestion
    source:
      test_commits: false
```

Since this repository doesn't have a package.xml it will fail our CI testing.

---

Sounds good.

Also, just FYI this non-ROS package is in support of the real ROS package (https://github.com/ifm/ifm3d-ros) and that'll be next on my list once the core driver/library is in the distribution.

Thanks for both your time and help.